src/lib/Helpers.coffee

```diff
@@ -347,7 +347,7 @@ exports.WirePattern = (component, config, proc) ->
               # FIXME probably need to skip this if payload is null
               closeGroupOnOuts payload
             # Disconnect
-            if _wp(scope).groupBuffers[port].length is 0 and payload is null
+            if _wp(scope).groupBuffers[port].length is 0
```

Comment:
With this PR, connect/disconnect are no longer sent as IP objects, and so this was needed to allow WirePattern to disconnect